libsolidity/interface/StandardCompiler.cpp

```diff
@@ -160,6 +161,39 @@ Json::Value collectEVMObject(eth::LinkerObject const& _object, string const* _so
 	return output;
 }
 
+Json::Value compileLLL(Json::Value const& _input)
+{
+	if (!_input["source"].isString())
```

Comment:
We've agreed on a call some time ago that the input should be the same format as for Solidity. This needs to be changed to reflect that.